app/styles/_util.scss

```diff
@@ -2,3 +2,11 @@
   pointer-events: none;
   opacity: 0.4;
 }
+
+.unread-label {
+    background-color: $color-bg-gray-light;
```

Comment:
Are we back to 4-space indents for CSS?

Reply:
Oops... We should add a linter for that.